src/scenes/Projects/queries.graphql

```diff
@@ -0,0 +1,68 @@
+query projects($input: ProjectListInput!) {
```

Comment:
Also I want to stop using "queries" for the filename. Let's make these files match what they are querying. `projects.graphql` is fine. That lets us go-to/open the file easier with our editors.